src/main/kotlin/com/openlattice/data/storage/PostgresQueries.kt

```diff
@@ -218,7 +217,7 @@ internal fun selectEntityKeyIdsWithCurrentVersionSubquerySql(
         }
     }
 
-    return "( SELECT $selectColumns FROM ${ENTITY_KEY_IDS.name} INNER JOIN $FILTERED_ENTITY_KEY_IDS USING($joinColumnsSql) WHERE true $entitiesClause $groupBy ) as $ENTITIES_TABLE_ALIAS"
+    return "( SELECT $selectColumns FROM entity_key_ids INNER JOIN $FILTERED_ENTITY_KEY_IDS USING($joinColumnsSql) WHERE true $entitiesClause $groupBy ) as $ENTITIES_TABLE_ALIAS"
```

Comment:
I think entity_key_ids replace should have been switched to IDS.name

Reply:
This class is appears to be a leaf node in a chain of deprecated/unused/commented code so I just switched it to the historical name for that variable. I can swap this class to IDS but I'm fairly sure it won't work/is unused

Reply:
Oh right. Good point, we should see if we can delete this.

Reply:
https://jira.openlattice.com/browse/LATTICE-2547